doc/releases/release-notes-2.4.rst

```diff
@@ -439,7 +439,7 @@ Drivers and Sensors
     application relies on any of the following Kconfig options, then
     it shall also enable the USB subsystem:
 
-    * :option:`CONFIG_OPENTHREAD_NCP_SPINEL_ON_UART_ACM`
+    * :option:`CONFIG_OPENTHREAD_COPROCESSOR_SPINEL_ON_UART_DEV_NAME`
```

Comment:
No need to modify 2.4 release notes, this PR won't get into 2.4 due to current feature freeze.

Reply:
True, but CI was complaining about an unknown option. Will fix with a rebase after release of 2.4.

Reply:
Rebased on current master and change in releasenotes removed.

Reply:
Unfortunately CI complains again:
```
Error: /home/runner/work/zephyr/zephyr/doc/_build/rst/doc/releases/release-notes-2.4.rst:579: WARNING: unknown option: CONFIG_OPENTHREAD_NCP_SPINEL_ON_UART_ACM
Error: Process completed with exit code 1.
```

Why is the current master checked against release notes for CONFIG_* variables?